app/src/main/java/org/jellyfin/androidtv/util/Utils.java

```diff
@@ -44,12 +46,12 @@ public static void showToast(Context context, int resourceId) {
         Toast.makeText(context, context.getString(resourceId), Toast.LENGTH_LONG).show();
     }
 
-    public static int convertDpToPixel(Context ctx, int dp) {
+    public static int convertDpToPixel(@Nullable Context ctx, int dp) {
         return convertDpToPixel(ctx, (float) dp);
     }
 
-    public static int convertDpToPixel(Context ctx, float dp) {
-        float density = ctx.getResources().getDisplayMetrics().density;
+    public static int convertDpToPixel(@Nullable Context ctx, float dp) {
+        float density = ctx == null ? 1 : ctx.getResources().getDisplayMetrics().density;
```

Comment:
Why is the density 1 when there is no context provided?

Reply:
I thought it would be the safest default value, but I'm open to suggestions. :smile:

Reply:
Change the context parameter to not-null. Places that don't send a context should be fixed instead